internal/controlplane/handlers_profile.go

```diff
@@ -551,6 +558,13 @@ func (s *Server) PatchProfile(ctx context.Context, ppr *minderv1.PatchProfileReq
 		return nil, status.Errorf(codes.Internal, "failed to get profile: %s", err)
 	}
 
+	// TEMPORARY HACK: Since we do not need to support this patch logic in the
+	// bundles yet, just reject any attempt to patch a profile from a bundle
+	// TODO: Move this patch logic to ProfileService
+	if oldProfile.SubscriptionID.Valid {
+		return nil, status.Errorf(codes.InvalidArgument, "cannot patch profile from bundle")
+	}
```

Comment:
Should this logic be added to `UpdateProfile` as well?

Reply:
The logic inside `ProfileService` takes care of this. Ideally, the patch and delete methods live inside that service as well - I will implement that in a future PR.